include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h

```diff
@@ -47,6 +47,12 @@ static const char* const kOrtSessionOptionsConfigSetDenormalAsZero = "session.se
 // Its default value is "0"
 static const char* const kOrtSessionOptionsDisableQuantQDQ = "session.disable_quant_qdq";
 
+// It controls whether to enable Double QDQ remover and Indentical Children Consolidation
```

Comment:
nit: check: "Indentical" is a misspelling of "Identical"

Reply:
Nice catch, thanks!